### PR TITLE
Add documentation of 'revokeSessions' parameter to appropriate APIs

### PR DESCRIPTION
**Pull request by edjohns-okta**
## Description:
- **Add documentation of 'revokeSessions' parameter to appropriate APIs** 

- **Is this PR related to a Monolith release?**  2022.12.0

### Resolves:

* [OKTA-553270](https://oktainc.atlassian.net/browse/OKTA-553270)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -3032,17 +3034,20 @@ curl -v -X POST \
...
-| newPassword  | New password for user                                   | Body       | [Password object](#password-object)  | TRUE     |
+| Parameter    | Description                                             | Param Type | DataType                             | Required | Default |
+| ------------ | ------------------------------------------------------- | ---------- | ------------------------------------ | -------- |---------|
+| id           | `id` of user                                            | URL        | String                               | TRUE     |         |
+| strict       | If true, validates against password minimum age policy  | Query      | String                               | FALSE    | FALSE   |
+| oldPassword  | Current password for user                               | Body       | [Password object](#password-object)  | TRUE     |         |
+| newPassword  | New password for user                                   | Body       | [Password object](#password-object)  | TRUE     |         |
+|revokeSessions | if true, revoke user sessions                          | Body       | boolean                              | FALSE    | FALSE  |
```
Comment (olegandreyev-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:3050]: nit: space

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 2]
```diff
@@ -1804,17 +1804,20 @@ Content-Type: application/json
 
 Changes a user's password by providing the existing password and the new password for authentication transactions with either the `PASSWORD_EXPIRED` or `PASSWORD_WARN` state
 
+This operation provides an option to revoke all the sessions of the specified user, except for the current session.
```
Comment (olegandreyev-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:1807]: doesn't this need to be all sessions as it's an admin operation?

**Review by olegandreyev-okta — APPROVED** [review]
left a comment and minor nit. otherwise LGTM

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md (hunk 1 above)
Comment (edjohns-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md]: Fixed.

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 3]
```diff
@@ -1804,17 +1804,20 @@ Content-Type: application/json
...
-| ----------- | ---------------------------------------------------------------- | ---------- | --------- | -------- |
-| newPassword | New password for user                                            | Body       | String    | TRUE     |
-| oldPassword | User's current password that is expired or about to expire       | Body       | String    | TRUE     |
-| stateToken  | [state token](#state-token) for the current transaction              | Body       | String    | TRUE     |
+| Parameter   | Description                                                      | Param Type | DataType  | Required | Default |
+| ----------- | ---------------------------------------------------------------- | ---------- | --------- | -------- |---------|
+| newPassword | New password for user                                            | Body       | String    | TRUE     |         |
+| oldPassword | User's current password that is expired or about to expire       | Body       | String    | TRUE     |         |
+| revokeSessions | if true, revoke user sessions                                 | Body       | boolean   | FALSE    | FALSE   |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:1819]:
```suggestion
| revokeSessions | When set to `true`, revokes all user sessions, except for the current session | Body       | boolean   | FALSE    | FALSE   |
```

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 4]
```diff
@@ -2614,6 +2614,7 @@ Generates a one-time token (OTT) that can be used to reset a user's password.  T
 
 This operation will transition the user to the status of `RECOVERY` and the user will not be able to login or initiate a forgot password flow until they complete the reset flow.
 
+This operation provides an option to delete all the user' sessions.  However, if the request is made in the context of a session owned by the specified user, that session will not be cleared.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:2617]:
```suggestion
This operation provides an option to delete all the user's sessions.  However, if the request is made in the context of a session owned by the specified user, that session isn't cleared.
```

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 5]
```diff
@@ -2623,6 +2624,7 @@ This operation will transition the user to the status of `RECOVERY` and the user
 | --------- | ------------------------------------------------ | ---------- | -------- | -------- | ------- |
 | id        | `id` of user                                     | URL        | String   | TRUE     |         |
 | sendEmail | Sends reset password email to the user if `true` | Query      | Boolean  | FALSE    | TRUE    |
+| revokeSessions | if true, revoke user sessions               | Query      | Boolean  | FALSE    | FALSE   |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:2627]:
```suggestion
| revokeSessions | When set to `true`, revokes all user sessions, except for the current session               | Query      | Boolean  | FALSE    | FALSE   |
```

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 6]
```diff
@@ -3032,17 +3034,20 @@ curl -v -X POST \
 
 Changes a user's password by validating the user's current password
 
+This operation provides an option to delete all the sessions of the specified user.  However, if the request is made in the context of a session owned by the specified user, that session will not be cleared.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:3037]:
```suggestion
This operation provides an option to delete all the sessions of the specified user.  However, if the request is made in the context of a session owned by the specified user, that session isn't cleared.
```

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 7]
```diff
@@ -3032,17 +3034,20 @@ curl -v -X POST \
...
-| newPassword  | New password for user                                   | Body       | [Password object](#password-object)  | TRUE     |
+| Parameter    | Description                                             | Param Type | DataType                             | Required | Default |
+| ------------ | ------------------------------------------------------- | ---------- | ------------------------------------ | -------- |---------|
+| id           | `id` of user                                            | URL        | String                               | TRUE     |         |
+| strict       | If true, validates against password minimum age policy  | Query      | String                               | FALSE    | FALSE   |
+| oldPassword  | Current password for user                               | Body       | [Password object](#password-object)  | TRUE     |         |
+| newPassword  | New password for user                                   | Body       | [Password object](#password-object)  | TRUE     |         |
+| revokeSessions | if true, revoke user sessions                         | Body       | boolean                              | FALSE    | FALSE  |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:3050]:
```suggestion
| revokeSessions | When set to `true`, revokes all user sessions, except for the current session                         | Body       | boolean                              | FALSE    | FALSE  |
```

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Comment by brianduffield-okta** [comment]
Merging to release branch

**Pull request merged**